src/main/native/glue/a/AaBox.cpp

```diff
@@ -43,6 +44,20 @@ JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains
     return result;
 }
 
+/*
+ * Class:     com_github_stephengold_joltjni_AaBox
+ * Method:    contains
+ * Signature: (JJ)Z
+ */
+JNIEXPORT jboolean JNICALL Java_com_github_stephengold_joltjni_AaBox_contains__JJ
```

Comment:
When you have two native methods with the same name, the Java compiler appends a signature suffix (in this case "__JJ" to distinguish them. This tends to introduce bugs (in this case, failure to rename the other method in the .cpp) so it's generally a good idea to make sure native methods don't have name conflicts like this.